src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStepTest.java

```diff
@@ -56,6 +57,7 @@
  * Test for {@link VariantLoaderStep}
  */
 @RunWith(SpringRunner.class)
+@ActiveProfiles("variant-annotation-mongo")
```

Comment:
Yeah... surprisingly it does.
VariantLoaderStep->Autowires AnnotationFlowOptional at AnnotationFlow -> Autowires AnnotationLoad.